docs/framework/unmanaged-api/profiling/icorprofilerinfo8-interface.md

```diff
@@ -21,4 +21,5 @@ A subclass of [ICorProfilerInfo7](../../../../docs/framework/unmanaged-api/profi
 **Header:** CorProf.idl, CorProf.h  
 **.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)]  
 ## See also
```

Comment:
```suggestion

## See also
```

Reply:
I saw a few of those, but resisted touching them :)
I'll add them to a batch though